repoman/pym/repoman/modules/scan/depend/_depend_checks.py

```diff
@@ -160,7 +161,12 @@ def _depend_checks(ebuild, pkg, portdb, qatracker, repo_metadata):
 			check_slotop(mydepstr, pkg.iuse.is_valid_flag,
 				badsyntax, mytype, qatracker, ebuild.relative_path)
 
+	dedup = collections.defaultdict(set)
 	for m, b in badsyntax:
+		if b in dedup[m]:
```

Comment:
Sorry, it seems I was not explicit enough. The if seems redundant to me as adding multiple times the same element to a `set` adds the item only once. This if would be needed in the `defaultdict` used a list though.